libs/extended/src/Network/AMQP/Extended.hs

```diff
@@ -119,3 +182,9 @@ logException l m (SomeException e) = do
   Log.err l $
     Log.msg m
       . Log.field "error" (displayException e)
+
+readCredsFromEnv :: IO (Text, Text)
+readCredsFromEnv = do
+  username <- Text.pack <$> getEnv "RABBITMQ_USERNAME"
+  password <- Text.pack <$> getEnv "RABBITMQ_PASSWORD"
+  pure (username, password)
```

Comment:
I like applicative syntax, but that is a matter of taste.

```suggestion
readCredsFromEnv =
  (,) <$> (Text.pack <$> getEnv "RABBITMQ_USERNAME") <*> (Text.pack <$> getEnv "RABBITMQ_PASSWORD")
```

Reply:
I added some new lines. Its applicatives all the way down :turtle: